### ISSUE TITLE
Access removed weekly to project 

### ISSUE THREAD

**Issue by CarolineDCT** [microsoft/clarity#457]
Member is added via email invite, project is added to their profile, 7 days later it is no longer there and they cannot access the project. (User has access to other projects but 2 of the projects keep disappearing of thier profile.) 

In the teams setting it still has this user listed as a member of the project and I have to remove and re invite them weekly.

**Comment by cvocvo** [comment]
Were you ever able to solve this? Our team just lost access to 3 of our projects and when we look in Webmaster and go to Clarity it acts like it's setting up a brand new project for the website?

**Comment by CarolineDCT** [comment]
No, I was never able to resolve this I just have to manually remove user and re invite them - most annoying as everyone else can access no problem and also they have access to other projects only some drop off.





________________________________

**Comment by ClaritySupport** [comment]
@CarolineDCT , To view a project, user need to sign in to Clarity with same email ID and AUTH provider which were used to create/added to the project. They won't be able to see project even if they use different AUTH provider to sign in to Clarity.

**Comment by CarolineDCT** [comment]
Thank you they are doing this we have checked



________________________________

**Comment by ClaritySupport** [comment]
@CarolineDCT ,Could you please share project IDs and user email IDs and AUTH providers they are using correctly but unable to view projects.

**Comment by cvocvo** [comment]
> Reference in

What's the best email to share this information? (Because it seems we may have the same or a similar issue.) I sent in an email with our information yesterday to `clarityms@microsoft.com` and haven't seen a response back. It seems like I should just make new projects, I guess?

**Comment by ClaritySupport** [comment]
@cvocvo , We did received and are checking with engineering team and will update you in email.

**Comment by MayuraKeskar1** [comment]
Hi @ClaritySupport , I had mailed ClarityMS@microsoft.com last month using my ID to recover my projects which do not show up in my project list even if I have signed in with Google Auth with my registered mail Id. Please let me know what is the issue??

**Comment by ClaritySupport** [comment]
Hello,

It seems issue with Google AUTH provider. It's ID got changed hence while authenticating it treated as new AUTH provider hence you are unable to view project.

We have responded to your individual email ID with the process.


Thanks,
Clarity Support Team
Visit our docs!: https://docs.microsoft.com/en-us/clarity
***@***.******@***.***>
***@***.***

Confidentiality note: This e-mail, and any attachment to it, contains privileged and confidential information intended only for the use of the individual(s) or entity named in the e-mail.  If the reader of the e-mail is not the intended recipient, or the employee or agent responsible for delivering it to the intended recipient, you are hereby notified that reading it is strictly prohibited.  If you have received this e-mail in error, please immediately return it to the sender and delete it from your system.
